doc/engineering-reference/src/integrated-solution-manager/zone-air-mass-flow-conservation.tex

```diff
@@ -75,11 +80,18 @@ \subsubsection{Allocation to Multiple Return Nodes}\label{allocation-to-multiple
 {\dot m_{R,i}} = {\dot m_{R,i}}
 \end{equation}
 
-The zone air mass flow conservation calculation limits the zone the return node air flow rate with air loop design supply to protect from large number the solution may throw as follows:
+In the zone air mass flow conservation calculation, the calculated zone total return flow rate is distributed to the zone return nodes proportional to the return nodes current mass flow rates as follows:
+
+\begin{equation}
+{\dot m_{R,i,}} = {\dot m_{R,i}} * \left(\frac{ReturnFlowSchedule*{\dot m_{R}}} {\sum\nolimits_{i} {{{\dot m}_{R,i}}}} \right) 
+\end{equation}
+
+The zone air mass flow conservation calculation also limits the zone return node air flow rate with air loop design supply to protect from large number the solution may throw as follows:
 
```

Comment:
Edited the text and added equation how the zone total return flow rate is distributed to a zone with multiple return nodes.